locations/hours.py

```diff
@@ -893,6 +893,13 @@ def set_closed(self, days: str | list[str]):
             self.day_hours.pop(day, None)
             self.days_closed.add(day)
 
+    def set_closed_missing_days(self):
+        for day in DAYS:
+            if self.day_hours.get(day, None):
+                pass
+            else:
+                self.days_closed.add(day)
+
```

Comment:
I would prefer to not include this change here. Can you remove it from the PR and make a separate PR that adds this feature along with tests? We can discuss there if we want to do this.

Reply:
Sure, I removed it